crates/ironrdp-displaycontrol/src/client.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{
+    pdu::{DisplayControlMonitorLayout, DisplayControlPdu, MonitorLayoutEntry},
+    CHANNEL_NAME,
+};
+use ironrdp_dvc::{encode_dvc_messages, DvcClientProcessor, DvcMessage, DvcProcessor};
+use ironrdp_pdu::PduResult;
+use ironrdp_svc::{impl_as_any, ChannelFlags, SvcMessage};
+use tracing::debug;
+
+/// A client for the Display Control Virtual Channel.
+pub struct DisplayControlClient {}
+
+impl_as_any!(DisplayControlClient);
+
+impl DvcProcessor for DisplayControlClient {
+    fn channel_name(&self) -> &str {
+        CHANNEL_NAME
+    }
+
+    fn start(&mut self, _channel_id: u32) -> PduResult<Vec<DvcMessage>> {
+        Ok(Vec::new())
+    }
+
+    fn process(&mut self, _channel_id: u32, payload: &[u8]) -> PduResult<Vec<DvcMessage>> {
+        // TODO: We can parse the payload here for completeness sake,
+        // in practice we don't need to do anything with the payload.
+        debug!("Got Display PDU of length: {}", payload.len());
```

Comment:
nitpick: Structured logging. See the `STYLE.md` file.
```suggestion
        debug!(length = payload.len(), "Got Display PDU");
```

Reply:
Solved in a better way in a pending PR: https://github.com/Devolutions/IronRDP/pull/424/files#diff-c1a0eb25e3d18b0feebe03529855499031e3be1368fc068e93f7afa91c37ccefR62-R67